xcdat/regridder/base.py

```diff
@@ -54,6 +54,9 @@ def preserve_bounds(
         else:
             if source_bnds.name in target:
                 logger.debug(f"Bounds {source_bnds.name!r} already present")
+            elif dim_name in ["X", "Y"]:
+                # the X / Y bounds are copied from the target grid above
+                continue
```

Comment:
I skip copying the `X` and `Y` bounds information, because I think they should always be populated in the logic above from the target grid. Is this an okay assumption? Regardless, I think the source grid horizontal bounds are no longer applicable?

Reply:
I guess one question here: can you regrid just `X` or `Y` (e.g., if I have [`time`, `lat`] datasets)? I think in this scenario you could still skip copying the source dataset `X` and `Y` bounds.